dynamic_rest/serializers.py

```diff
@@ -510,7 +510,7 @@ class WithDynamicModelSerializerMixin(WithDynamicSerializerMixin):
 
     @classmethod
     def get_model(cls):
-        return cls.Meta.model
+        return getattr(cls.Meta, 'model', None)
```

Comment:
What's the manifestation of this bug? If a ModelSerializer has model, things will start to fail down the line..